lib/module.js

```diff
@@ -250,7 +250,7 @@ Module._resolveLookupPaths = function(request, parent) {
 
   // make sure require('./path') and require('path') get distinct ids, even
   // when called from the toplevel js file
-  if (parentIdPath === '.' && id.indexOf('/') === -1) {
+  if (parentIdPath === '.' && id.includes('/')) {
```

Comment:
resolved thru soft reset and override in push -f... so this diff is now outdated. thx for spotting, @yosuke-furukawa